### ISSUE TITLE
Sync aqi widget background with aqi condition

### ISSUE THREAD

**Issue by KianBadie** [caltechlibrary/air-sampling#6]
Add functionality to have the aqi widget background reflect the current condition of the overall aqi.

**Comment by KianBadie** [comment]
I have attached a gif below of the current possible backgrounds for the aqi widget (if you give it a second, it will cycle through all the options). These colors are based off of the current stop colors on the aqi meter we have below the aqi widget.

Not all colors meet contrast ratios for accessible readings. I just stuck to the colors from the meter for now. I wanted to get opinions first before modifying. Do you prefer the meter and the aqi widget colors stay in sync? If so, we should modify both. Or are you ok with the colors being different?

Please ignore the unsmooth display of the aqi meter gradient. That is a result of the recording software I used.

![demo](https://user-images.githubusercontent.com/18221058/204678368-9deaf422-b5f3-41af-ab02-2c836adf9a3c.gif)

**Comment by KianBadie** [comment]
In addition, here are the colors for the gradient with hard stops. Putting it here as a reference in the case that you wanted to try out different colors.

![meter](https://user-images.githubusercontent.com/18221058/204681967-53e02924-4ef4-499d-8a06-20f0b220dc74.PNG)

**Comment by crounjd** [comment]
I do like the discreet colors.  I do not like the color bar,  it is very subdued.  Can we find something with bright colors?
![image](https://user-images.githubusercontent.com/20955490/204901491-c44eb940-441e-4e0b-af01-1ee00ac7bff5.png)

**Comment by crounjd** [comment]
I suppose as with  the image I posted above, we could switch the color of the font from black to white, when moving into the darker color regime, if contrast is an issue.

**Comment by KianBadie** [comment]
@crounjd I think that is a great idea. So if I am understanding you correctly: use the colors in the chart you shared for the color bar, use the discreet colors for the overall aqi display (maybe we can make them the same colors in the chart but in a lighter hue?), and switch between black/white text where necessary?

**Comment by KianBadie** [comment]
@crounjd I updated the site with some new colors. I tried to keep the discreet aspect for the aqi background. Because of that, the maroon color was the only color that the white font worked with. I also used hard stops instead of a gradient for the color bar, as well as using the exact colors is the aqi doc. Feel free to let me know if any thoughts or opinions come up for this feature!

![demo](https://user-images.githubusercontent.com/18221058/205118047-faee7f24-6853-4bf1-8907-607d3e993ad5.gif)

**Comment by crounjd** [comment]
Sounds good. I like the colors in the color bar.  Some of the colors displayed in the box above are different than in the color bar?  Seems most apparent on orange and red?

**Comment by KianBadie** [comment]
@crounjd That is correct, they are actually all different. I interpreted "discreet" as lighter colors for the aqi widget. My apologies, would you like all the colors to be the same (as in the aqi document)?

**Comment by crounjd** [comment]
I think so, unless there is some reason not to do this?  Otherwise, at least to me, it looks off.

**Comment by KianBadie** [comment]
@crounjd There wasn't a particular reason for the lighter colors I chose. I just misunderstood and was also weary how the hard greens, reds, etc would look so I opted for the lighter colors. I can go ahead and switch those colors to how they look in the docs!

**Comment by KianBadie** [comment]
@crounjd I switched the colors to match the docs exactly. This means I also switched the blue text we had to be black (except for purple/maroon backgrounds where the text is white). If you would like the blue text back, I can add it back in. It works for green, yellow, and orange background. Red would be the only background that needs black text to meet contrast ratios. And of course the purple/maroon backgrounds would need white text to be readable.

These colors are also reflected in small aqi boxes in the pollutant dropdowns on the site.
